test/images/agnhost/dns/dns.go

```diff
@@ -28,7 +28,9 @@ const etcHostsFile = "/etc/hosts"
 // nameserver DNS_CLUSTER_IP
 // search test-dns.svc.cluster.local svc.cluster.local cluster.local q53aahaikqaehcai3ylfqdtc5b.bx.internal.cloudapp.net
 // options ndots:5
-func getDNSSuffixList() []string {
+
+// GetDNSSuffixList reads DNS config file and returns the list of configured DNS suffixes
+func GetDNSSuffixList() []string {
```

Comment:
FWIW using strings.Fields() would be safer WRT repeated whitespace

Reply:
The only thing that is changed here is that function becomes exported having uppercase first letter. So, I do not understand the point of the comment.

Reply:
exporting it and starting to use it as a utility method for other packages expands the scope of the code and increases the importance that it be robust/correct

Reply:
Strictly speaking this function gets a list of "search paths" and if I were to find "" in that list (which you would if there was a double-space) I would be very surprised.

Is there a test-case for this function?  I'm not going to demand you go write one, unless you want to, but it probably should have a test including values with duplicate whitespace

Reply:
@liggitt , @thockin I agree that it would be good to fix this thing, but I disagree that it must be part of this particular change that has one concrete goal - fixing concrete bug. And the change you propose me doing is not related to the bugfix at all.

Then, about unit tests, the fact that I changed the name of the function and unit tests passed means that there are either no tests or they were not run. In anyway, it doesn't get tested on regular basis. So, it has much more TODOs which must be done, but definitely not as part of this change, IMHO.